software/heyform.yml

```
@@ -0,0 +1,11 @@
+name: HeyForm
+website_url: https://heyform.net
+description: Form builder that allows anyone to create engaging conversational forms for surveys, questionnaires, quizzes, and polls.
+licenses:
+  - AGPL-3.0
+platforms:
+  - Docker
+  - Nodejs
```

```suggestion
  - Docker
```

As Docker is the only supported/documented installation method